torch/nn/modules/module.py

```diff
@@ -544,6 +641,17 @@ def register_forward_hook(self, hook):
         self._forward_hooks[handle.id] = hook
         return handle
 
+    @classmethod
+    def clear_global_hooks(cls):
```

Comment:
Why do we want this? The user already has the handles to clear the states if he wants to. This could potentially clear out other hooks that the user is not aware of.

Reply:
I use it mostly in the testing code to remove the global state between tests.
Should pretending `_` to it be ok? I can also recreate the dicts in the teardown code as well.